docs/azure/integrations-overview.md

```diff
@@ -0,0 +1,360 @@
+---
+title: .NET Aspire Azure integrations overview
+description: Overview of the Azure integrations available in the .NET Aspire.
+ms.date: 12/11/2024
+uid: azure-integrations-overview
+---
+
+# .NET Aspire Azure integrations overview
+
+[Azure](/azure) is the most popular cloud platform for building and deploying [.NET applications](/dotnet/azure).  The Azure SDK for .NET allows for easy management and use of Azure services. .NET Aspire provides a set of integrations with Azure services. This article details some common aspects of all Azure integrations in .NET Aspire and aims to help you understand how to use them.
+
+## Provisioning
```

Comment:
This term is overloaded. I would be more descriptive with what these libraries are for. The industry standard term for these libraries are a "CDK" (cloud development kit). I would also use Infrastructure as code somewhere in here.

Reply:
I was told I shouldn't use CDK, since this is different than the industries usage of CDK.

Reply:
@tg-msft mentioned to me in the past to not refer to Azure.Provisioning as the "CDK".

Reply:
Lets not conflate locaal provisioning with the provisioning libraries then. We need a better title.

Reply:
Added a section specifically to address this